### DISCUSSION TITLE
Tab completion in the issue/PR comment editor appears broken

### DISCUSSION THREAD

**Discussion by scjody** [community/community#21757] [New to GitHub]
<p>We have several teams in our organization (plotly), and I often tag <span class="mention">@plotly</span>/devops in issue or PR comments. Up until about a week ago I was able to do this by typing <span class="mention">@plotly</span>/dev[TAB] but now it autocompletes to <span class="mention">@plotly</span>/dash. Even <span class="mention">@plotly</span>/devop[TAB] ends up as <span class="mention">@plotly</span>/dash. Thinking it might be trying to match on the description of the team, I added a space to the <span class="mention">@plotly</span>/dash description but it still completes to <span class="mention">@plotly</span>/dash.</p>
<p>Is there anything I can do to restore the tab completion behaviour I expect?</p>
<p><img src="https://user-images.githubusercontent.com/1136329/181066560-17a0a89d-7546-4dff-bb39-94638cf575c3.png" alt="Screen Shot 2019-04-11 at 10.54.45.png" data-base62-sha1="yPTYqKORcv4SX83m9LD1hO7R9rV" width="653" height="126"></p>

**Comment by lee-dohm** [comment]
<p>Thanks for the feedback.</p>
<p>What happens if you type <code>@plotly/do</code>? It appears to be matching using a fuzzy search pattern. In other words, you can type any sequence of characters as long as they’re in the right order.</p>
<p>As for the current behavior, I’ll pass along the feedback to the developers. I can’t give a timeline of when it might get addressed, but I’ll let the right people know about the experience.</p>
<p>Thanks again for your help!</p>

**Comment by scjody** [comment]
<p>Thanks for the reply!</p>
<p>“<span class="mention">@plotly</span>/do” matches “<span class="mention">@plotly</span>/dash-core Dash Core Team” first. “<span class="mention">@plotly</span>/devops” is 5th on that list.</p>
<p><img src="https://user-images.githubusercontent.com/1136329/181066563-5e4090b7-f2c4-4df7-9cf9-0355abe9db21.png" alt="Screen Shot 2019-04-16 at 14.38.12.png" data-base62-sha1="2jFL3av9UeqmnX0ewpF65eApEzB" width="586" height="204"></p>
<p>Thanks for passing on my feedback. Fuzzy searching makes sense, but given that an exact match exists I’d expect that to be the first thing shown.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p><a class="mention" href="/scjody">@scjody</a> It appears that this has been fixed, or at least improved. Give it a try!</p>

**Comment by scjody** [comment]
<p>Looks good now. Thanks for the update, and thanks to whoever made the change!</p>
<p><img src="https://user-images.githubusercontent.com/1136329/181066565-f9c75160-1d68-4b2e-be56-b8d9a56c4659.png" alt="Screen Shot 2019-04-30 at 15.45.34.png" data-base62-sha1="cSsekT5b76Uf0UmlfAnpTAAx0s5" width="563" height="108"></p>
